### PR TITLE
Pgo 40 helm monitoring

### PR DESCRIPTION
**Pull request by benjaminjb**

### CODE REVIEW

File: helm/monitoring/dashboards/postgresql_details.json [hunk 1]
```diff
@@ -0,0 +1,2148 @@
...
+      "fieldConfig": {
+        "defaults": {
+          "color": {
+            "mode": "thresholds"
+          },
+          "custom": {},
+          "links": [
+            {
+              "title": "pgBackrest",
+              "url": "/d/2fcFZ6PGk/pgbackrest?${__all_variables}"
```
Comment (benjaminjb) [inline helm/monitoring/dashboards/postgresql_details.json:88]: Might be wrong: https://github.com/CrunchyData/pgmonitor/compare/v4.8.1..v4.8.0

Comment (cbandy) [reply helm/monitoring/dashboards/postgresql_details.json:88]: Wanna compare this direction, I think: https://github.com/CrunchyData/pgmonitor/compare/v4.8.0..v4.8.1

File: kustomize/monitoring/deploy-grafana.yaml
[the remark this answers is not in the document]
Comment (benjaminjb) [reply kustomize/monitoring/deploy-grafana.yaml:1]: With kustomize, iirc, if you create object named X, then update your files and create object named Y, you'll have both X and Y -- we would have to tell people to delete the X objects. (That said, I think everything I changed the name of was something that wouldn't actually interfere: a cluster role, a cluster-role-binding, a service account.)

Things that change internally (like all the configmaps that had `.yml` changed to `.yaml`) would be updated in situ, and as long as you updated the cm and the deployment using it, you'd be fine.

NOW: you asked this question on a kustomize file, but I think it's worth talking about the Helm scenario: some people have bashed together their own Helm charts with different named objects. What will happen when they upgrade? The objects that are no longer managed by the Helm chart will be removed and the new objects created. (So if we renamed services, they just get replaced.)

(Note: that said, I just noticed a problem in our Helm chart here: deployment matchLabels shouldn't be tied to specific chart versions.)

**Comment by benjaminjb** [comment]
Opening https://github.com/CrunchyData/postgres-operator-examples/pull/219

**Pull request closed without merging**